### DISCUSSION TITLE
Client logo URL is not in REST API (SOLVED)

### DISCUSSION THREAD

**Discussion by senyahnoj** [keycloak/keycloak#12674] [Ideas]
On Keycloak 16.1.1 there are 3 new useful URL fields which are added to a client record

![Screenshot from 2022-06-23 10-02-10](https://user-images.githubusercontent.com/5585851/175264935-46f5bd3a-cf74-4e99-8eec-f341792deb55.png)

* Logo URL
* Policy URL
* Terms of service URL

However, none of these appear to be present in the REST API https://www.keycloak.org/docs-api/16.1/rest-api/index.html#_clientrepresentation

In the interests of automation, it would be good to be able to set these 3 URLS in

* POST /{realm}/clients
* PUT /{realm}/clients/{id}

And retrieve these 3 URLS in

* GET /{realm}/clients
* GET /{realm}/clients/{id}

**Comment by senyahnoj** [comment]
https://github.com/keycloak/keycloak/issues/12676

**Reply by senyahnoj** [reply]
My mistake. They are present in the REST API and appear as attributes of the client.

**Comment by cgeorgilakis** [comment]
It would be good to have again these three fields in Account Console as in Keycloak 16.1.1 . Unfortunately, a PR remove these three fields from Account Console.
Could Keycloak team proceed with reviewing [my related PR](https://github.com/keycloak/keycloak/pull/11837)?
